.gitignore

```diff
@@ -15,6 +15,7 @@ bin/
 *.iml
 *.iws
 out/
+.vscode/
```

Comment:
Meh

Reply:
yes, I don't know why suddenly I have that locally.
Perhaps an upgrade of vscode introduced that